R/FilterStateChoices.R

```diff
@@ -459,37 +461,32 @@ ChoicesFilterState <- R6::R6Class( # nolint
             }
           })
 
+          observeEvent(private$disabled(), {
```

Comment:
```suggestion
          observeEvent(private$is_disabled(), {
```

It looks like all other class (FilterStateDate, FilterStateDatetime) is using `private$is_disabled()` method rather than the `private$disabled` property. I propose we do the same for consistency.